girder_annotation/test_annotation/test_annotations.py

```diff
@@ -416,6 +416,55 @@ def testGetNextVersionValue(self):
         val3 = Annotationelement().getNextVersionValue()
         assert val3 > val2
 
+    def testOverlayBounds(self, server, admin, fsAssetstore):
+        file = utilities.uploadExternalFile('sample_image.ptif', admin, fsAssetstore)
+        itemId = str(file['itemId'])
+
+        # test no transform
+        lowx, highx, lowy, highy = Annotationelement()._overlayBounds({
+            'type': 'imageoverlay', 'girderId': itemId
+        })
+        assert lowx == 0
+        assert lowy == 0
+        assert highx == 58368
+        assert highy == 12288
+
+        # test offset
+        lowx, highx, lowy, highy = Annotationelement()._overlayBounds({
+            'type': 'imageoverlay', 'girderId': itemId,
+            'transform': {'xoffset': 500, 'yoffset': 1000}
+        })
+        assert lowx == 500
+        assert lowy == 1000
+        assert highx == 58868
+        assert highy == 13288
+
+        # test affine matrix, scale to 50%
+        lowx, highx, lowy, highy = Annotationelement()._overlayBounds({
+            'type': 'imageoverlay', 'girderId': itemId,
+            'transform': {
+                'matrix': [[0.5, 0], [0, 0.5]]
+            }
+        })
+        assert lowx == 0
+        assert lowy == 0
+        assert highx == 58368 / 2
+        assert highy == 12288 / 2
+
+        # test transform and scaling
+        lowx, highx, lowy, highy = Annotationelement()._overlayBounds({
+            'type': 'imageoverlay', 'girderId': itemId,
+            'transform': {
+                'xoffset': 500,
+                'yoffset': 1000,
+                'matrix': [[0.5, 0], [0, 0.5]]
+            }
+        })
+        assert lowx == 500
+        assert lowy == 1000
+        assert highx == (58368 / 2) + 500
+        assert highy == (12288 / 2) + 1000
+
```

Comment:
Great.  Can you also add a test that invokes the `_boundingBox` method to test that it invokes the `_overlayBounds` method appropriately?  It doesn't have to test the various transform options, just the `'type': 'imageoverlay'` pass-through.